docs/translation-guide.md

```diff
@@ -14,7 +14,7 @@ Our apps and website are translated through Weblate: A libre web-based translati
 
 Once you’ve selected a project, you can provide suggestions for strings that have not yet been translated, or suggest changes to strings that have already been translated. These suggestions will be evaluated by a translation team member and they will choose the most appropriate one. For a more info on Weblate, you can refer to its [documentation](https://docs.weblate.org/en/weblate-3.0.1/user/index.html).
 
-By default, you're only able to suggest translations. If you would like to get permissions to save translations, [join the translators Slack](https://translators-slack.elementary.io/) and message an admin your Weblate username and what language you want to translate.
+By default, you're only able to suggest translations. If you would like to get permissions to save translations, [join the translators Slack](https://join.slack.com/t/ele-l10n/shared_invite/enQtNDkyOTA1MTEzMTQxLWNiMjhmODdlYzE4N2MzMDQ1N2RkNzY2YWYwZjc2NWY4YWUxZmNlODljNjJmN2EzMWQwMDY4ZmFlNzcwNTc0YTU) and message an admin your Weblate username and what language you want to translate.
```

Comment:
Slack invite links do not work the way we want them too. They either have an expiration date (slack keeps changing this), or they have an invite limit. Both of those require maintenance to keep links accurate and working. We are currently looking at alternatives and will update the DNS to work when we find a solution. For more info you can follow the issue here: 

https://github.com/elementary/website/issues/2714#issuecomment-841729349